collect_app/src/main/res/drawable/main_menu_button_background.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="?colorAccent">
```

Comment:
This looks wrong to me playing around with it: I see a blue ripple. I think this should be a light grey right? Maybe `colorOnSurfaceVariant`. I'm wondering how much we could steal from/be using the `Widget.Material3.Button.ElevatedButton` (and `.Filled` style for the `StartNewFormButton`) here instead of needing our own styles and backgrounds.

Reply:
I think it should be grey too, and it would be great to use as much of the M3 styling as possible. I didn't know if that would be part of this story or if we would update the overall button styling later because I assumed the blue ripple was applied throughout the app.

Reply:
> I'm wondering how much we could steal from/be using the Widget.Material3.Button.ElevatedButton (and .Filled style for the StartNewFormButton) here instead of needing our own styles and backgrounds. For instance, even if we can't use the styles themselves we could use standard dimens/values like the @color/m3_button_ripple_color_selector and @color/m3_button_background_color_selector that Widget.Material3.Button uses.

Reply:
>This looks wrong to me playing around with it: I see a blue ripple. I think this should be a light grey right?

I used the gray color.

>I'm wondering how much we could steal from/be using the Widget.Material3.Button.ElevatedButton (and .Filled style for the StartNewFormButton) here instead of needing our own styles and backgrounds. For instance, even if we can't use the styles themselves we could use standard dimens/values like the @color/m3_button_ripple_color_selector and @color/m3_button_background_color_selector that Widget.Material3.Button uses.

Yes, I can't use the style itself because it's a custom _layout,_ not a button. When it comes to using some particular resources that are used by that style it's also not possible because they are private and can be used internally only.